### ISSUE TITLE
Is there any possible way to split a multi-speaker model into individual speaker models?

### ISSUE THREAD

**Issue by engineergun** [TensorSpeech/TensorFlowTTS#516]
Again and again thanks for this awesome repository. I trained a multi-speaker fastspeech2 model using fastspeech2_libritts example with 4 speakers and it works fine with the speaker_ids parameter. But, I am wondering is there any possible way to split them into individual speaker models?

**Comment by machineko** [comment]
You can just retrain it for few epochs on loss that will eradicate the generation of audio of any other speaker_id embedding 📦

**Comment by engineergun** [comment]
@machineko Thanks a lot :)
